CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [Version]
```

Comment:
Can you change this to `Unreleased` to prevent merge conflicts?

https://github.com/iTwin/iTwinUI-react/blob/0aaa0c31dfbc10b88888cf52156d326d2a78da29/CONTRIBUTING.md#L175-L181

Reply:
Changed.

Reply:
There will still be merge conflicts. Should be `Unreleased` not `[Unreleased]` 😆

Reply:
Changed.